src/app/app.module.ts

```diff
@@ -76,7 +67,7 @@ import { FaqAccordionComponent } from './faqs/faq-accordion/faq-accordion.compon
     MaterialModule,
     FlexLayoutModule,
   ],
-  providers: [KToolsService],
+  providers: [KToolsService, TicketService],
```

Comment:
I think it would be best to leave services out of providers arrays and just use `@Injectable({ providedIn: 'root' })`